core/modules/taxonomy/js/taxonomy.term.js

```diff
@@ -0,0 +1,44 @@
+(function ($) {
+
+Backdrop.behaviors.Term = {
+  // Provide the vertical tab summaries.
+  attach: function (context) {
+    var $context = $(context);
+
+    // Relation settings.
+    $context.find('#edit-relations').backdropSetSummary(function(context) {
+      var vals = [];
+      var info = Backdrop.t('No relations.');
+
+      // Get a list of all selected options and concatenate with a comma.
+      var parents = $.map(
+        $context.find('select[name="parent[]"] option:selected'),
+        function(element) {
+          // Remove leading hyphens on indented terms.
+          return $(element).text().replace(/^(\-)+/, '');
+        })
+        .join(', ');
+
+      if (parents) {
+        info = Backdrop.t('Parents: @parents', { '@parent': parents });
```

Comment:
Pretty sure there's an error here with parents/parent. Should be: `info = Backdrop.t('Parents: @parents', { '@parents': parents });`

Reply:
We could improve this further also by using formatPlural() instead of t(). I'll make another adjustment.